xml5ever/src/tree_builder/types.rs

```diff
@@ -16,15 +16,15 @@ use crate::tokenizer::{Doctype, Pi, Tag};
 
 #[derive(PartialEq, Eq, Copy, Clone, Debug)]
 pub enum XmlPhase {
-    StartPhase,
-    MainPhase,
-    EndPhase,
+    Start,
+    Main,
+    End,
 }
 
 /// A subset/refinement of `tokenizer::XToken`.  Everything else is handled
 /// specially at the beginning of `process_token`.
 #[derive(PartialEq, Eq, Clone, Debug)]
-pub enum Token {
+pub enum TokenWrapper {
```

Comment:
clippy complained, I think Wrapper makes sense, as XTokens get wrapped by it.
`pub` so might need version bump